optaplanner-core/src/test/java/org/optaplanner/core/impl/testdata/domain/scenario/simplequeens/TestdataQueenEntity.java

```diff
@@ -0,0 +1,41 @@
+package org.optaplanner.core.impl.testdata.domain.scenario.simplequeens;
+
+import org.optaplanner.core.api.domain.entity.PlanningEntity;
+import org.optaplanner.core.api.domain.variable.PlanningVariable;
+import org.optaplanner.core.impl.testdata.domain.TestdataEntity;
+import org.optaplanner.core.impl.testdata.domain.scenario.simplequeens.solution.TestdataQueenDifficultyWeightFactory;
+import org.optaplanner.core.impl.testdata.domain.scenario.simplequeens.solution.TestdataRowStrengthWeightFactory;
+
+@PlanningEntity(difficultyWeightFactoryClass = TestdataQueenDifficultyWeightFactory.class)
+public class TestdataQueenEntity extends TestdataEntity {
```

Comment:
I understand. On the other hand, there shouldn't be many opportunities for maintenance. Tests needn't use up-to-date queen scenario